packages/neuron-ui/src/components/SpecialAssetList/index.tsx

```diff
@@ -100,7 +101,11 @@ const SpecialAssetList = () => {
   }, [closeNewAccount, closeExistAccount, setMigrateCell])
 
   const {
-    app: { epoch, globalDialog },
+    app: {
+      epoch,
+      globalDialog,
+      feeRateStatics: { suggestFeeRate = MEDIUM_FEE_RATE },
```

Comment:
IMO, the default value can be set on initialization, it's repeated multiple times